editor/src/components/navigator/navigator-item/component-picker.tsx

```diff
@@ -59,20 +59,97 @@ export function componentPickerOptionTestId(componentName: string, variant?: str
 }
 
 export const ComponentPicker = React.memo((props: ComponentPickerProps) => {
+  const { onItemClick } = props
+  const [selectedComponentKey, setSelectedComponentKey] = React.useState<string | null>(null)
   const [filter, setFilter] = React.useState<string>('')
+  const menuRef = React.useRef<HTMLDivElement | null>(null)
 
-  const allComponentsToShow: InsertMenuItemGroup[] = []
+  const changeFilter = useCallback(
+    (newFilter: string) => {
+      setFilter(newFilter)
+    },
```

Comment:
Since `setFilter` just takes a string anyways, I'm not sure you need to define the wrapping arrow here.

ie. `const changeFilter = useCallback(setFilter, [setFilter])`

Reply:
You're right, it's a leftover from an earlier version where I also triggered highlighting. Will change (I can just put `setFilter` directly in the prop)